src/odemis/driver/tmcm.py

```diff
@@ -1538,6 +1595,8 @@ def moveRel(self, shift):
 
         # Check if the distance is big enough to make sense
         for an, v in shift.items():
+            if an in self._name_to_do_axis:
+                raise NotImplementedError("Ignoring relative move on digital output axis %s." % an)
```

Comment:
That's a bit weird message -> "Relative move on digital output axis not supported (requested on axis %s)".